solc/CommandLineParser.cpp

```diff
@@ -47,6 +47,7 @@ static string const g_strErrorRecovery = "error-recovery";
 static string const g_strEVM = "evm";
 static string const g_strEVMVersion = "evm-version";
 static string const g_strEwasm = "ewasm";
+static string const g_strEOFVersion = "eof-version";
```

Comment:
Maybe we should go with `--experimental-eof-version` as we did with `--via-ir`?

Reply:
Fine with me. Undocumented and up for change anyways.